database_lookup_search.py

```diff
@@ -0,0 +1,150 @@
+
+    tk.Label(details_frame, text="Social Security Number:", font=bold_label_font).grid(column=0, row=6, sticky="e", padx=padx, pady=pady)
+    tk.Label(details_frame, text=patient[5], font=normal_text_font).grid(column=1, row=6, sticky="w", padx=padx, pady=pady)
+
+    tk.Label(details_frame, text="Biological Gender:", font=bold_label_font).grid(column=0, row=7, sticky="e", padx=padx, pady=pady)
+    tk.Label(details_frame, text=patient[6], font=normal_text_font).grid(column=1, row=7, sticky="w", padx=padx, pady=pady)
+
+    tk.Label(details_frame, text="Religion:", font=bold_label_font).grid(column=0, row=8, sticky="e", padx=padx, pady=pady)
+    tk.Label(details_frame, text=patient[7], font=normal_text_font).grid(column=1, row=8, sticky="w", padx=padx, pady=pady)
+
+    tk.Label(details_frame, text="Language:", font=bold_label_font).grid(column=0, row=9, sticky="e", padx=padx, pady=pady)
+    tk.Label(details_frame, text=patient[9], font=normal_text_font).grid(column=1, row=9, sticky="w", padx=padx, pady=pady)
+
+    tk.Label(details_frame, text="VOCA Classifications:", font=bold_label_font).grid(column=0, row=10, sticky="e", padx=padx, pady=pady)
+    tk.Label(details_frame, text=patient[10], font=normal_text_font).grid(column=1, row=10, sticky="w", padx=padx, pady=pady)
+
+    tk.Label(details_frame, text="Comments:", font=bold_label_font).grid(column=2, row=0, sticky="e", padx=padx, pady=pady)
+    tk.Label(details_frame, text=patient[11], font=normal_text_font, wraplength=500).grid(column=3, row=0, sticky="w", padx=padx, pady=pady)
+
+    tk.Label(details_frame, text="Prior Convictions:", font=bold_label_font).grid(column=2, row=1, sticky="e", padx=padx, pady=pady)
+    tk.Label(details_frame, text="Yes" if patient[12] else "No", font=normal_text_font).grid(column=3, row=1, sticky="w", padx=padx, pady=pady)
+
+    tk.Label(details_frame, text="Convicted of Crimes Against Children:", font=bold_label_font).grid(column=2, row=2, sticky="e", padx=padx, pady=pady)
+    tk.Label(details_frame, text="Yes" if patient[13] else "No", font=normal_text_font).grid(column=3, row=2, sticky="w", padx=padx, pady=pady)
+
+    tk.Label(details_frame, text="Sexual Offender:", font=bold_label_font).grid(column=2, row=3, sticky="e", padx=padx, pady=pady)
+    tk.Label(details_frame, text="Yes" if patient[14] else "No", font=normal_text_font).grid(column=3, row=3, sticky="w", padx=padx, pady=pady)
+
+    tk.Label(details_frame, text="Sexual Predator:", font=bold_label_font).grid(column=2, row=4, sticky="e", padx=padx, pady=pady)
+    tk.Label(details_frame, text="Yes" if patient[15] else "No", font=normal_text_font).grid(column=3, row=4, sticky="w", padx=padx, pady=pady)
+
+#to filter patients based on search
+def search_patients(event=None):
+    global filtered_patients
+    config = load_config()
+
+    search_query = search_entry.get().lower()
+    filtered_patients = []
+
+    try:
```

Comment:
Can just use the [connect.py](https://github.com/ZaneLesley/NCATrak-Mock-System/blob/main/database/connect.py) file to facilitate this, instead of connecting like this many times.